src/structures/scoreboard.ts

```diff
@@ -45,33 +45,31 @@ export default class Scoreboard {
      */
     getWinnerMessage(guildID: string): string {
         let winnerStr = "";
-
-        if (this.firstPlace.length === 1) {
-            return i18n.translate(guildID, "misc.inGame.winMessage", {
-                playerName: this.firstPlace[0].getDisplayedName(
-                    false,
-                    false,
-                    false,
-                ),
-            });
-        }
-
         for (let i = 0; i < this.firstPlace.length; i++) {
-            winnerStr += this.firstPlace[i].getName();
+            winnerStr += this.firstPlace[i].getDisplayedName(
+                false,
+                false,
+                false,
+            );
             if (i === this.firstPlace.length - 1) {
                 // Last entry -- append just the username
-                winnerStr += " ";
+                winnerStr += "";
             } else if (i === this.firstPlace.length - 2) {
                 // Second last entry -- use "and"
-                winnerStr += " and ";
+                const and = i18n.translate(guildID, "misc.inGame.and");
```

Comment:
its never safe to assume you can just translate one word in the middle of a sentence and assume its grammatically correct in other languages

Reply:
https://www.i18next.com/translation-function/formatting#list